src/components/landing/CustomerLogoCloud.js

```diff
@@ -58,14 +58,12 @@ const LOGOS = [{
 }];
 
 /* eslint-disable jsx-a11y/alt-text */
-const LogoItem = ({ src, ...rest }) => (
-  <div className="col-span-1 flex justify-center">
-    <picture>
-      <source srcSet={src.webp} type="image/webp" />
-      <source srcSet={src.png} type="image/png" />
-      <img src={src.png} {...rest} className="h-8 sm:h-10" />
-    </picture>
-  </div>
+export const LogoItem = ({ src, ...rest }) => (
+  <picture>
+    <source srcSet={src.webp} type="image/webp" />
+    <source srcSet={src.png} type="image/png" />
+    <img src={src.png} {...rest} />
+  </picture>
```

Comment:
These changes allow me to re-use this component in the sidebar of the pricing pages.